src/main/java/net/mcreator/element/converter/ConverterRegistry.java

```diff
@@ -43,7 +43,7 @@
 import net.mcreator.element.converter.fv27.ProcedureShootArrowFixer;
 import net.mcreator.element.converter.fv28.FoodToItemConverter;
 import net.mcreator.element.converter.fv29.CommandParameterBlockFixer;
-import net.mcreator.element.converter.fv30.BlockRequiresCorrectToolConverter;
+import net.mcreator.element.converter.fv30.*;

```

Comment:
FV 30 is taken by `requiresCorrectTool` fixer (moreover, it's part of today's snapshot), your converter is not gonna run properly.

Reply:
Why wouldn't my converter run properly? Also there can be more than 1 fixer per FV bruh

Reply:
Name me at least one such case.

Reply:
FV11, FV14, FV21

Reply:
@KlemenDEV do you think I should move to converter FV31?

Reply:
Both fv11 converters fix coordinates used by GUI-type elements, all fv14 converters update luminance parameter on different mod elements, both fv21 converters are needed because of the variable system refactoring, whereas special information format fixers have nothing common with fv30 converter.